local-cli/generator-android/templates/src/app/BUCK

```diff
@@ -0,0 +1,68 @@
+import re
+
+# See [Docs](https://buckbuild.com/) to learn what BUCK is.
+# To run your application with BUCK you need:
+# - install BUCK
+# - `npm start` - to start the packager
+# - `cd android`
+# - `cp ~/.android/debug.keystore keystores/debug.keystore`
+# - `./gradlew :app:copyDownloadableDepsToLibs` - make all Gradle compile dependencies available to BUCK
+# - `buck install -r android-app` - compile, install and run application
+#
+
+lib_deps = []
+for jarfile in glob(['libs/*.jar']):
+  name = 'jars__' + re.sub(r'^.*/([^/]+)\.jar$', r'\1', jarfile)
+  lib_deps.append(':' + name)
+  prebuilt_jar(
+    name = name,
```

Comment:
Maybe a silly idea but is it possible to declare a `prebuilt_jar` without the name?
Can you make an array of the `prebuilt_jar` objects and pass it to `app-code` as deps? No idea if it works.

Or just use a name such as 'jars__' + jarfile so you don't have to transform it?